openreview/workflows/workflows.py

```diff
@@ -110,11 +110,11 @@ def set_workflows_group(self):
             )
         )
 
-    def set_reviewers_only_request(self):
+    def set_conference_review_request(self):
 
         super_id = self.super_id
         support_group_id = self.support_group_id
-        conference_venue_invitation_id = f'{support_group_id}/Venue_Request/-/Reviewers_Only'
```

Comment:
@melisabok this should have only changed the venue request name and dropped /Support